Dockerfile

```diff
@@ -1,38 +1,47 @@
-
-
-FROM ubuntu:21.04 as build
+FROM ubuntu:20.04 as build
 
 ENV LANG C.UTF-8
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN dpkg --add-architecture i386 && \
-    apt-get update && \
+RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y \
-        binutils:i386 \
-        gcc-10:i386 \
-        g++-10:i386 \
-        python3.10 \
-        python \
+        binutils \
+        gcc-10 \
+        g++-10 \
+	p7zip-full \
+        python3.9 \
         make \
         cmake \
+        curl \
         git \
         lld \
-        libsdl2-dev:i386 \
-        zlib1g-dev:i386 \
-        libbz2-dev:i386 \
-        libpng-dev:i386 \
-        libgles2-mesa-dev && \
-    ln -sf /usr/bin/python3.10 /usr/bin/python3 && \
-    ln -s /usr/bin/gcc-10 /usr/bin/gcc && \
-    ln -s /usr/bin/gcc-10 /usr/bin/cc && \
+        libsdl2-dev \
```

Comment:
is this just being installed for the dependencies?

Reply:
yes, for the dependencies. easier than listing all of them for install.